user-guides/data-farming/how-to-estimate-apy.md

```diff
@@ -0,0 +1,39 @@
+---
+description: >-
+  Learn how to use basic math and a simple spreadsheet to estimate your
+  Passive Rewards APY and get deeper into Active Rewards APY.
+---
+
+<figure><img src="../../.gitbook/assets/gif/mafs.gif" alt=""><figcaption><p>K.I.S.S.</p></figcaption></figure>
+
+# veOCEAN and your APY
+
+Rewards are earned by users that hold and use their veOCEAN to help the protocol improve and grow.  
+
+First, make sure that you [understand the basics of APY](../../rewards/df-veocean.md#veocean-and-your-apy) and how veOCEAN works.
+
+Now that we got that out of the way, let's work through the examples and keep it as simple as possible.
+
+### Estimating Passive APY
+
+To make it easier to estimate your APY, [we have created a simple spreadsheet](https://docs.google.com/spreadsheets/d/1zzuW5pBbX6j6hZL_XtJDtSR2-rDHa_LGOEwgoQ4D8lk/edit?usp=sharing) that let's you easily estimate your Passive APY.
+
+We have provided 2 sheets as an example of locking-up 10,000 OCEAN for:
+1. A 4-year investment period
+2. A 6-mo investment period
+
+The above are simplified examples meant for everyone to understand. They are naive investment strategies and are meant to provide you, the reader, with some examples to build upon. You can use these as a reference to create your own plan, so feel free to make a copy of this spreadsheet and customize it to your needs and wants.
```

Comment:
Big note here: The algorithm is built in such a way that it distributes good incentives for locking ocean tokens > 2years

Reply:
This is written in quite a few places, but I have also linked at the top so users make sure they understand the basics of APY.

```
To receive competitve short-term APYs, veOCEAN requires users to make long-term commitments.

First, make sure that you [understand the basics of APY](../../rewards/df-veocean.md#veocean-and-your-apy) and how veOCEAN works.
```

But, based on your feedback, I re-emphasized your point here.